src/esw/brushless.py

```diff
@@ -173,7 +180,12 @@ async def _connect(self) -> None:
                 ),
                 timeout=self.MOTEUS_RESPONSE_TIME_INDICATING_DISCONNECTED_S,
             )
-            self._check_has_error(state.values[moteus.Register.FAULT])
+            if state is None or not hasattr(state, "values") or moteus.Register.FAULT not in state.values:
+                self._check_has_error(99)
+                return
+            else:
```

Comment:
Remove the early return and just change to an if and else statement

Reply:
Also this code is duplicated (this fx is _connect while the other is _send_command) with only a few small changes, so if possible try to reduce code duplication